### DISCUSSION TITLE
publish config

### DISCUSSION THREAD

**Discussion by svenboogaart** [spatie/laravel-sitemap#405] [Q&A]
When I try to to run the following command

    php artisan vendor:publish --provider="Spatie\Sitemap\SitemapServiceProvider" --tag=config

I get "Unable to locate publishable resources."

I just installed it by running 

    composer require spatie/laravel-sitemap

**Comment by SZubanov** [comment]
Hey!
If this is still relevant, then you can manually copy the file from vendor
vendor/spatie/laravel-sitemap/config/sitemap.php
